### PR TITLE
Build script for simpleitk

### PR DESCRIPTION
**Pull request by ManirajDeivendran**

### CODE REVIEW

File: s/simpleitk/simpleitk_master_rhel_8.3.sh [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+yum update -y
+yum install -y cmake make gcc-c++ wget git libpng-devel patch
+
+# Clone and build source
+git clone https://github.com/SimpleITK/SimpleITK
+
+# Updated SLICImageFilter(minimum) test tolerance to safely 
+# ignore the failures. Refer 
+# (https://github.com/SimpleITK/SimpleITK/issues/1360) for more information.
+PATCH_LINE=$(awk '/^__PATCH_BEGIN__/ { print NR + 1; exit 0; }' $0)
```
Comment (seth-priya) [inline s/simpleitk/simpleitk_master_rhel_8.3.sh:28]: can we move the patch into a separate patch file please?

File: a/apache-flink/apache-flink_master_rhel_8.3.sh [hunk 2]
```diff
@@ -18,23 +18,68 @@ VERSION="master"
...
+cp otj-pg-embedded-0.13.3.jar ~/.m2/repository/com/opentable/components/otj-pg-embedded/0.13.3/
+cd ..
+
+# Create /var/run/postgresql and change ownership to current non-root user
+USER=`whoami`
+mkdir -p /var/run/postgresql/
+sudo chown $USER:$USER /var/run/postgresql/
+
+# Updated pom.xml with known failures (Tests which is same as x86_64)
+wget https://github.com/ManirajDeivendran/build-scripts/blob/master/a/apache-flink/apache-flink_ignore_test_fail_as_x86.patch
```
Comment (seth-priya) [inline a/apache-flink/apache-flink_master_rhel_8.3.sh:79]: I understand the failures are common to x86 and ppc64le, so lets not mention "x86" in the name, it can be confusing

File: a/apache-flink/apache-flink_master_rhel_8.3.sh [hunk 3]
```diff
@@ -18,23 +18,68 @@ VERSION="master"
...
 git checkout $VERSION
 
 # Updated com.google.protobuf:protoc from 3.5.1-->3.7.0
 sed -i 's/3.5.1/3.7.0/g' flink-formats/flink-parquet/pom.xml
 
-#Compile and build package using threads
+# Compile and build package using threads
 mvn clean package -T 6 -DskipTests -Dfast
+
+# Resolve "No postgres binary for ppc64le" error
```
Comment (seth-priya) [inline a/apache-flink/apache-flink_master_rhel_8.3.sh:48]: can you add more comments to explain what you are doing below?

File: a/apache-flink/apache-flink_master_rhel_8.3.sh [hunk 4]
```diff
@@ -18,23 +18,68 @@ VERSION="master"
...
-wget https://www-us.apache.org/dist/maven/maven-3/3.2.5/binaries/apache-maven-3.2.5-bin.tar.gz --no-check-certificate --quiet
-tar xzf apache-maven-3.2.5-bin.tar.gz
+# Install postgresql build dependencies
+sudo yum install -y https://download.postgresql.org/pub/repos/yum/reporpms/EL-8-ppc64le/pgdg-redhat-repo-latest.noarch.rpm
+sudo yum install -y postgresql10-libs
+sudo yum install -y postgresql10 postgresql10-server postgresql10-contrib
 
-export M2_HOME=`pwd`/apache-maven-3.2.5
-export PATH=`pwd`/apache-maven-3.2.5/bin:${PATH}
+# Install maven v3.6.3
```
Comment (seth-priya) [inline a/apache-flink/apache-flink_master_rhel_8.3.sh:29]: Should we make the maven version configurable?

**Review by seth-priya — CHANGES_REQUESTED** [review]

File: a/apache-flink/apache-flink_master_rhel_8.3.sh (hunk 3 above)
Comment (ManirajDeivendran) [reply a/apache-flink/apache-flink_master_rhel_8.3.sh]: Resolved. Updated the explanation.

File: a/apache-flink/apache-flink_master_rhel_8.3.sh (hunk 4 above)
Comment (ManirajDeivendran) [reply a/apache-flink/apache-flink_master_rhel_8.3.sh]: Created ENV variable to maintain the maven version.

**Review by ManirajDeivendran — COMMENTED** [review]
Fixed review comments.

**Review by seth-priya — APPROVED** [review]
Looks good. please squash the commits.
